telemetry/data_feature_usage_test.go

```diff
@@ -576,6 +576,36 @@ func TestAddIndexAccelerationAndMDL(t *testing.T) {
 	require.Equal(t, true, usage.DDLUsageCounter.MetadataLockUsed)
 }
 
+func TestDistReorgUsage(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	usage, err := telemetry.GetFeatureUsage(tk.Session())
+	require.Equal(t, int64(0), usage.DDLUsageCounter.DistReorgUsed)
+	require.NoError(t, err)
+
+	allow := variable.DDLEnableDistributeReorg.Load()
+	require.Equal(t, false, allow)
```

Comment:
We are not sure what the value is, depending on what you set it up to when you run the test, so you have to decide

Reply:
In 6.6， this system variable is default false right?

Reply:
Yes, but in CI we will run it with `distributereorg` is true and false.